virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,25 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough 
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+Your cluster and virtual machines can access PCI host devices or mediated hardware devices through a virtual GPU (vGPU) passthrough configuration. Configuring a vGPU passthrough allows you to:
+
+* Access your underlying hardware's GPU, achieving full-performance benefits in your virtual machine.
+
+* Passthrough supported hardware, such as a graphics card.
+
+* Enable resource-intensive operations such as dual-booting.
```

Comment:
`Enable resource-intensive operations such as data processing`

Reply:
Would this be OK?

* Enable resource-intensive I/O operations.